src/shogun/io/fmt/fmt.h

```diff
@@ -3,8 +3,10 @@
 
 #if defined(USE_EXTERNAL_SPDLOG)
 #include <spdlog/fmt/fmt.h>
+#include <spdlog/fmt/bundled/ostream.h>
 #else
 #include <shogun/third_party/spdlog/fmt/fmt.h>
+#include <shogun/third_party/spdlog/fmt/bundled/ostream.h>
```

Comment:
@vigsterkr is it ok to add the ostream.h? Makes it easier to format stuff with ftm, e.g. format("{}", object_instance), where object instance has operator<< overloaded

Reply:
well lets have it in for the time being and we'll see later... if this helps to have a faster dev cycle, i dont see why we shouldnt have it :)